<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmevent</class>
 <widget class="QWidget" name="frmevent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统日志</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#frm{
background-image: url(:/image/main2.jpg);
}

QLabel{
color:#E7ECF0;
background-color:rgba(0,0,0,0);
border-style:none;
}

QPushButton#btnFirst{
font: 12pt &quot;微软雅黑&quot;;
color:#E7ECF0;
background-color:rgba(0,0,0,0);
border-style:none;
}

QPushButton#btnPre{
font: 12pt &quot;微软雅黑&quot;;
color:#E7ECF0;
background-color:rgba(0,0,0,0);
border-style:none;
}

QPushButton#btnNext{
font: 12pt &quot;微软雅黑&quot;;
color:#E7ECF0;
background-color:rgba(0,0,0,0);
border-style:none;
}

QPushButton#btnLast{
font: 12pt &quot;微软雅黑&quot;;
color:#E7ECF0;
background-color:rgba(0,0,0,0);
border-style:none;
}

QTableView{
	background-color:rgba(255,255,255,80);
	border-style:none;
}</string>
  </property>
  <widget class="QWidget" name="frm" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <widget class="QLabel" name="labInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>421</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>共有 0 条数据   共 0 页   当前第 0 页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNext">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>440</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="icon">
     <iconset resource="main.qrc">
      <normaloff>:/image/next.png</normaloff>:/image/next.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPre">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>440</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="icon">
     <iconset resource="main.qrc">
      <normaloff>:/image/pre.png</normaloff>:/image/pre.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableView" name="tableEvent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>91</y>
      <width>781</width>
      <height>335</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLast">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>440</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="icon">
     <iconset resource="main.qrc">
      <normaloff>:/image/last.png</normaloff>:/image/last.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>起始时间：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>124</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFirst">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>440</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="icon">
     <iconset resource="main.qrc">
      <normaloff>:/image/first.png</normaloff>:/image/first.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>结束时间：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOk">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>124</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>399</x>
      <y>10</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>日</string>
    </property>
   </widget>
   <widget class="QComboBox" name="end_hour">
    <property name="geometry">
     <rect>
      <x>429</x>
      <y>50</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>10</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>年</string>
    </property>
   </widget>
   <widget class="QComboBox" name="end_day">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>50</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>10</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>时</string>
    </property>
   </widget>
   <widget class="QComboBox" name="end_year">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>50</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="start_hour">
    <property name="geometry">
     <rect>
      <x>429</x>
      <y>10</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="start_year">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>10</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>50</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>年</string>
    </property>
   </widget>
   <widget class="QComboBox" name="start_month">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>50</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>时</string>
    </property>
   </widget>
   <widget class="QComboBox" name="end_month">
    <property name="geometry">
     <rect>
      <x>209</x>
      <y>50</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>50</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>月</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>399</x>
      <y>50</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>日</string>
    </property>
   </widget>
   <widget class="QComboBox" name="start_day">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>10</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>月</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
